scripts/generate_abi.py

```diff
@@ -0,0 +1,97 @@
+import json
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import click
+from psutil import Popen
+
+COMPILE_COMMAND = """
+git clone git@github.com:beamer-bridge/beamer.git {beamer_path} &&
```

Comment:
We can use `--no-checkout` option here to avoid git checking out the main branch by default, since we are anyway going to checkout our own commit.